Yoast/wordpress-seo
# Call right cache clear method

## Summary


This PR can be summarized in the following changelog entry:

* N/A - In the hotfix for 12.8.1, the method we used as callback is renamed

## Relevant technical choices:

* Note: please merge the release branch to trunk after merging this pull request.

## Test instructions

This PR can be tested by following these steps:

* We just applied the rename here as well.
* Go to the social settings, change some frontpage values
* View the source of the frontpage and make sure the changes are applied.

## UI changes

## Documentation

## Quality assurance
